regression/cbmc/Pointer_Arithmetic18/main.c

```diff
@@ -0,0 +1,11 @@
+#define MB 0x00100000
+#define BASE (15 * MB)
+#define OFFSET (1 * MB)
+
+main()
+{
+  char *base = BASE;
```

Comment:
I'm a little concerned by this.  As I understand CBMC's memory model we do not support literal addresses.  I don't think we are required to do so by the standard either.

Reply:
I'd say we half-support them... And I'd really like to see that fixed as well, might just happen along the way ;-)